frontend/js/annotation-tool.js

```diff
@@ -731,9 +731,16 @@ define(["jquery",
 
                 bookData.forEach(function (arr) {
                     Object.keys(arr).forEach(function (key) {
-                        var value = arr[key] === null ? '' : arr[key];
+                        let value = arr[key] === null ? '' : arr[key];
 
-                        objectMaxLength[key] = Math.max(objectMaxLength[key], value.length);
+                        let len = 0;
+                        switch(typeof value) {
+                          case "number": len = 10; break;
+                          case "string": len = value.length; break;
+                          case "object": if(value instanceof Date) len = 10; break;
+                        }
```

Comment:
```suggestion
                        switch (typeof value) {
                        case "number":
                            len = 10;
                            break;
                        case "string":
                            len = value.length;
                            break;
                        case "object":
                            if (value instanceof Date) len = 10;
                            break;
                        }
```

Some formatting here, but also can I ask how you came up with these cases? This seems a bit arbitrary. :thinking: And btw. what happens in the `"object"` case when it's not a `Date`?

And I might be wrong here, but the library probably puts something like `value.toString()` into the resulting file, right? So can't we just use `value.toString().length`? :thinking:

Reply:
Indeed very arbitrary, due to me not wanting to deal with proper column width calculation. 

Unfortunately just using `value.toString()` can often be insufficient. E.g. for the ID you would get
![Bildschirmfoto vom 2021-03-24 11-11-36](https://user-images.githubusercontent.com/14070005/112294834-8ab30000-8c93-11eb-9fd4-c6a46b97b288.png)
instead of 
![Bildschirmfoto vom 2021-03-24 11-15-59](https://user-images.githubusercontent.com/14070005/112294881-9acadf80-8c93-11eb-819e-9fa5d82d5a02.png)

I updated the PR with a different len calculation based on your suggestion. It is still arbitrary, but at least it is more concise. Please check if it is more to your liking.

Reply:
I feared it wouldn't be that trivial. :thinking: 

Alright, I made #474 to track the issue, but it's not a priority, and also not in the scope of this "hotfix" PR. This is good enough for now. :slightly_smiling_face: